extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs

```diff
@@ -331,7 +326,7 @@ private async Task<ProviderOperationResult> HandleActionWhenFormInInitialState(A
             // if OnAction's state is reviewForm, then the user has reviewed the form and Dev Home has started the creation process.
             // we'll show the same form to the user in Dev Homes summary page.
             shouldEndSession = true;
-            operationResult = await GetForReviewFormAdaptiveCardAsync(OriginalUserInputJson);
+            operationResult = await GetForReviewFormAdaptiveCardAsync(GetAndValidateInput(OriginalUserInputJson));
```

Comment:
Consider a new variable as it would be better to read and debug.

Reply:
sure  I can do that